test/e2e/specs/editor/various/navigable-toolbar.spec.js

```diff
@@ -240,6 +239,68 @@ test.describe( 'Block Toolbar', () => {
 		await editor.setIsFixedToolbar( false );
 		await pageUtils.setBrowserViewport( 'large' );
 	} );
+
+	test( 'Focus should remain on mover when moving blocks', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		// On default floating toolbar
+		await editor.insertBlock( {
+			name: 'core/paragraph',
+			attributes: { content: 'Paragraph 1' },
+		} );
+		await editor.insertBlock( {
+			name: 'core/paragraph',
+			attributes: { content: 'Paragraph 2' },
+		} );
+		await editor.insertBlock( {
+			name: 'core/paragraph',
+			attributes: { content: 'Paragraph 3' },
+		} );
+		await pageUtils.pressKeys( 'shift+Tab' );
+		// check focus is within the block toolbar
+		const blockToolbarParagraphButton = page.getByRole( 'button', {
+			name: 'Paragraph',
+			exact: true,
+		} );
+		await expect( blockToolbarParagraphButton ).toBeFocused();
+		// Go to Move Up Button
+		await pageUtils.pressKeys( 'ArrowRight' );
+		const blockToolbarMoveUpButton = page.getByRole( 'button', {
+			name: 'Move up',
+			exact: true,
+		} );
+
+		// Make sure it's in an acvite state for now
+		await expect( blockToolbarMoveUpButton ).not.toHaveAttribute(
+			'aria-disabled',
+			'true'
+		);
+
+		await expect( blockToolbarMoveUpButton ).toBeFocused();
+		await pageUtils.pressKeys( 'Enter' );
+		await expect( blockToolbarMoveUpButton ).toBeFocused();
+		await pageUtils.pressKeys( 'Enter' );
+		await expect( blockToolbarMoveUpButton ).toBeFocused();
+		await expect( blockToolbarMoveUpButton ).toHaveAttribute(
+			'aria-disabled',
+			'true'
+		);
```

Comment:
```suggestion
		await expect( blockToolbarMoveUpButton ).toBeDisabled();
```

Reply:
This is the Playwright assertion, which checks both - https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-disabled.

> Element is disabled if it has "disabled" attribute or is disabled via 'aria-disabled'.

Reply:
Ugh. Can't believe I missed that. Sorry! I'll add another little PR. Thanks for correcting me!